packages/alchemy/src/provider/__tests__/light-account.test.ts

```diff
@@ -39,18 +39,6 @@ describe("Alchemy Provider Tests", () => {
     `);
   });
 
-  it("should correctly sign the message", async () => {
```

Comment:
no longer a unit test because signMessage has to check the LA version via `getLightAccountVersion` which checks on-chain impl address? do you wanna mock here?

Reply:
it exists in the actual packages/account instead